app/src/main/java/com/example/recyclerviewdemo/Person.java

```diff
@@ -0,0 +1,29 @@
+package com.example.recyclerviewdemo;
+
+public class Person {
+    private int mImage;
+    private String mTen;
```

Comment:
```suggestion
    private String mName;
```Nên là tiếng anh hết